src/js/dex/orderbook.component.js

```diff
@@ -38,6 +48,10 @@
     function OrderbookController() {
         var ctrl = this;
 
+        ctrl.cancel = function (obj) {
+            ctrl.cancelOrder(obj.order);
+        };
+
         ctrl.$onChanges = function () {
             if (ctrl.type !== 'user') {
```

Comment:
Fixed this. Also, I split the component into two, for orderbooks and for user orders.